### PR TITLE
Address feedback from reviewer-2

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
cf. https://github.com/openjournals/joss-reviews/issues/7240

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+*   Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`. 
+    If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing. 
+*   Create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("brief-description-of-change")`. At a minimum, please avoid submitting PRs from your fork's `main` branch` as this can make the review process more complicated.
+
+*   Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser.
+    The title of your PR should briefly describe the change.
+    The body of your PR should contain `Fixes #issue-number`.
+
+*  For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first header). Follow the style described in <https://style.tidyverse.org/news.html>.
```
Comment (MichaelChirico) [inline .github/CONTRIBUTING.md:35]: > add a bullet to the top of `NEWS.md` (i.e. just below the first header).

This feels a bit misleading. I think the intention is "add a bullet in the (under development) section", but typically we don't add bullets at the very top of the file. Suggestion:

```suggestion
*  For user-facing changes, add a bullet appropriately in the top section of `NEWS.md` (i.e. below the first header). Follow the style described in <https://style.tidyverse.org/news.html>.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Code style
+
+*   New code should follow the tidyverse [style guide](https://style.tidyverse.org). 
+    You can use the [styler](https://CRAN.R-project.org/package=styler) package to apply these styles, but please don't restyle code that has nothing to do with your PR.
```
Comment (MichaelChirico) [inline .github/CONTRIBUTING.md:40]:
```suggestion
    You can use the [styler](https://CRAN.R-project.org/package=styler) package to apply these styles.
```

I don't adhere to "never touch anything not strictly related to your PR", I rather like the boyscout rule (IIRC you've cited this yourself on this repo). Anyway, the code in {lintr} is already pretty well {styler}-aligned, so I don't think we need to emphasize anything in this section.

File: paper/paper.Rmd [hunk 3]
```diff
@@ -65,7 +65,7 @@ withr::local_options(list(
...
+In computer programming, "linting" is the process of analyzing the source code to identify possible programming and stylistic problems (https://en.wiktionary.org/wiki/linting) and a linter is a tool used for linting. A linter analyzes code to identify potential errors, stylistic issues, or deviations from coding standards. It helps ensure consistency, readability, and best practices by flagging common mistakes, such as syntax errors, unused variables, or improper formatting. Linters are essential for improving code quality, preventing bugs, and maintaining a clean codebase, especially in collaborative development environments [@enwiki:1218663830]. `{lintr}` is an open-source package that provides linters for the R programming language, which is an interpreted, dynamically-typed programming language [@base2023], and is used by a wide range of researchers and data scientists. `{lintr}` can thus act as a valuable tool for R users to help improve the quality and reliability of their code.
```
Comment (MichaelChirico) [inline paper/paper.Rmd:68]: should this be a citation as opposed to a parenthesized link?

Comment (IndrajeetPatil) [reply paper/paper.Rmd]: Yes, it'll be better as a citation. I have created a new bibtex entry.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+### Pull request process
+
+*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("r-lib/lintr", fork = TRUE)`.
+
+*   Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`. If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing. 
+
+*   Create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("brief-description-of-change")`. At a minimum, please avoid submitting PRs from your fork's `main` branch` as this can make the review process more complicated.
+
+*   Make your changes, commit them to Git, and create a PR using `usethis::pr_push()`. Follow the prompts in your browser to complete the process. Use a concise title for your PR that summarizes the change, and include `Fixes #issue-number` in the PR description.
```
Comment (MichaelChirico) [inline .github/CONTRIBUTING.md:25]:
```suggestion
*   Make your changes, commit them to Git, and create a PR using `usethis::pr_push()`. Follow the prompts in your browser to complete the process. Use a concise title for your PR that summarizes the change, and include `Fixes #issue-number` in the PR _description_. Doing so will automatically closed the linked issue when the PR is merged.
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+### Pull request process
+
+*   Fork the package and clone onto your computer. If you haven't done this before, we recommend using `usethis::create_from_github("r-lib/lintr", fork = TRUE)`.
+
+*   Install all development dependencies with `devtools::install_dev_deps()`, and then make sure the package passes R CMD check by running `devtools::check()`. If R CMD check doesn't pass cleanly, it's a good idea to ask for help before continuing. 
+
+*   Create a Git branch for your pull request (PR). We recommend using `usethis::pr_init("brief-description-of-change")`. At a minimum, please avoid submitting PRs from your fork's `main` branch` as this can make the review process more complicated.
+
+*   Make your changes, commit them to Git, and create a PR using `usethis::pr_push()`. Follow the prompts in your browser to complete the process. Use a concise title for your PR that summarizes the change, and include `Fixes #issue-number` in the PR _description_. Doing so will automatically close the linked issue when the PR is merged.
```
Comment (MichaelChirico) [inline .github/CONTRIBUTING.md:25]:
```suggestion
*   Make your changes, commit them to Git, and create a PR using `usethis::pr_push()`. Follow the prompts in your browser to complete the process. Use a concise title for your PR that summarizes the change, and include `Fixes #issue-number` in the PR _description_. Doing so will automatically close the linked issue when the PR is merged. For complicated changes, add a textual overview of what your PR does in the description. Consider breaking up large PRs into a chain of more digestible+focused smaller PRs.
```

**Review by MichaelChirico — APPROVED** [review]
Thank you!

**Pull request merged**